src/carousel/carousel.scss

```diff
@@ -112,10 +114,8 @@
     }
   }
 
-  &-animation-button {
-    order: 1;
-
-    &[data-pressed="true"] {
+  &-animation-button[data-pressed='true'] {
+    @include themed {
       outline: 1px solid var(--iui-color-foreground-primary);
```

Comment:
If you're using the themed mixin, then change this CSS variable to use `t`

Reply:
Maybe also change the other instance in line 49

Reply:
Oh dang, I didn't even notice that was a CSS variable and not one of our `t()` variables, my mistake.   Both 119 & 49 are using the `t()` now.